.env.example

```diff
@@ -1,3 +1,3 @@
-REACT_APP_KEYCLOAK_realm = Tolkevarav DEV
```

Comment:
The reason why I removed those is that it gives out information to others, since repos are public. e.g. someone sees this url and decides to make a ddos attack etc.